frontend/src/features/upload/components/FirstContentAreaHeader.tsx

```
@@ -0,0 +1,44 @@
+      <div className='flex items-center gap-6'>
+        {uploadTab === 'file' && (
+          <div className='flex items-center gap-1 text-gray-400 select-none'>
+            <div>0</div>
+            <div>/</div>
+            <div>10MB</div>
+          </div>
+        )}
+        <button
+          className='flex cursor-pointer gap-[0.375rem] rounded-lg bg-gray-50 px-[1.125rem] py-3'
+          onClick={() => nextStep?.('2')}
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)